pkg/ipamclaimswebhook/podmutator.go

```diff
@@ -190,6 +194,23 @@ func (a *IPAMClaimsValet) Handle(ctx context.Context, request admission.Request)
 	return admission.Allowed("carry on")
 }
 
+func validateDefaultMultusNetworkRequest(pod *corev1.Pod, primaryUDNInterfaceName string) error {
+	defaultNet, err := udn.GetK8sPodDefaultNetworkSelection(
+		pod.Annotations[config.MultusDefaultNetAnnotation],
```

Comment:
let's check in the caller is the annotation is actually present instead of passing an empty string down.